Changes

```diff
@@ -29,6 +29,14 @@ Next version (4.05.0):
 - GPR#891: Use -fno-builtin-memcmp when building runtime with gcc.
   (Leo White)
 
+- GPR#70:
+  The runtime can now be shut down gracefully by means of the new
+  caml_shutdown and caml_startup_pooled functions. The new 'c' flag in
+  OCAMLRUNPARAM enables shutting the runtime properly on process exit.
+  (Max Mouratov, review and discussion by Damien Doligez, Gabriel Scherer,
+  Mark Shinwell, Thomas Braibant, Stephen Dolan, Pierre Chambart,
+  François Bobot, and Jacques Garrigue)
+
```

Comment:
Sorry - I spotted the git diff artefact, but not the obvious here! Wrong GPR number :smile: Ideally, the paragraph should flow on the same line (i.e. `GPR#71: The runtime can now be...`)